data/transit/route/subway.json

```diff
@@ -342,6 +356,20 @@
         "route": "subway"
       }
     },
+    {
+      "displayName": "TCL",
+      "id": "tcl-c48f12",
+      "locationSet": {"include": ["fr"]},
+      "tags": {
+        "network": "TCL",
+        "network:wikidata": "Q8700",
+        "network:wikipedia": "fr:Transports en commun lyonnais",
+        "operator": "Keolis Lyon",
+        "operator:wikidata": "Q61456136",
+        "operator:wikipedia": "fr:Keolis",
```

Comment:
Same comment as above - remove the `operator:wikipedia` if it isn't for "Keolis Lyon"